geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/data/AccumuloFeatureSource.scala

```diff
@@ -228,7 +232,7 @@ class DelegatingResourceInfo(source: SimpleFeatureSource) extends ResourceInfo {
 
   import scala.collection.JavaConversions._
 
-  private val keywords = Collections.unmodifiableSet(Set("features", getName))
+  private val keywords: java.util.Set[String] = new java.util.HashSet[String](List("features", getName))
```

Comment:
does it work like:
`Collections.unmodifiableSet(Set("features", getName) ++ source.getSchema.getKeywords)`
?
++ will take two unmodifiable collections and create a new one from them.